src/styles/blocks/card.scss

```diff
@@ -0,0 +1,122 @@
+.card-menu {
+  --count-column: 1;
+
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(var(--count-column) 200px);
+  grid-template-rows: repeat(2, 406px);
+  gap: 48px 46px;
+
+  @media (min-width: 488px) {
+    --count-column: 2;
+  }
+
+  @media (min-width: 768px) {
+    --count-column: 3;
+  }
+
+  @media (min-width: 1024px) {
+    --count-column: 4;
+  }
+}
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  border-radius: 5px;
+  background-color: 1px solid #f3f3f3;
+  transition: transform 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+
+    .card_title {
+      color: #34568b;
+    }
+  }
+
+  &__title {
+    display: flex;
+    font-weight: 500;
+    font-size: 12px;
+    margin-top: 40px;
+    margin-bottom: 0;
+    line-height: 18px;
+    color: #060b35;
+  }
+
+  &__info {
+    width: 98px;
+    height: 14px;
+    font-weight: 400;
+    font-size: 10px;
+    margin-right: 68px;
+    margin-top: 4px;
+    margin-bottom: 16px;
+    line-height: 14px;
+    color: #616070;
+  }
+
+  &_imagep {
```

Comment:
The class name `&_imagep` is incorrect. It should be `&__image` to maintain consistency with the BEM naming convention.